common/components/componentsBySection/FindProjects/ProjectCard.jsx

```diff
@@ -87,19 +87,19 @@ class ProjectCard extends React.PureComponent<Props> {
         <ul>
         {this.props.project.location &&
           <li>
-            <i className={GlyphStyles.MapMarker}></i>
+            <i className={GlyphStyles.MapMarker + ' fa-fw'}></i>
```

Comment:
"fa-fw" should live in a constant, so folks know what it means.

Reply:
Changed to a const and named it so it's meaningful. I agree that longer term we should change the glyphs constructor to accept an options object or something like you describe.